rootski_api/src/rootski/config/config.py

```diff
@@ -279,6 +279,11 @@ def customise_sources(
                 file_secret_settings,  # ???
             )
 
+    def __init__(self, **kwargs):
```

Comment:
Gotcha, looks like this is what's causing branches from trunk to fail? I had to look up what kwargs was haha. At first, I just said "kwargs", "keyword args" makes a lot more sense.

Reply:
Yep! Yeah, `kwargs` is the idiomatic identifier of this concept in python. I do agree that `keyword_args` would be more readable.